zipline/data/minute_bars.py

```diff
@@ -441,6 +441,13 @@ def pad(self, sid, date):
         assert new_last_date == date, "new_last_date={0} != date={1}".format(
             new_last_date, date)
 
+    def set_bcolz_attrs(self, sid, **kwargs):
```

Comment:
It's in support of an adjustment factor, so that we can store a greater range of floats without truncating any decimal places.

Agreed on the renaming and test coverage.